cmd/limactl/list.go

```diff
@@ -14,10 +14,10 @@ import (
 
 func newListCommand() *cobra.Command {
 	listCommand := &cobra.Command{
-		Use:               "list",
+		Use:               "list [flags] [INSTANCE]...",
 		Aliases:           []string{"ls"},
 		Short:             "List instances of Lima.",
-		Args:              cobra.NoArgs,
+		Args:              cobra.ArbitraryArgs,
 		RunE:              listAction,
 		ValidArgsFunction: cobra.NoFileCompletions,
```

Comment:
Shouldn't this complete instance names?

```suggestion
		ValidArgsFunction: listBashComplete,
```

and

```go
func listBashComplete(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
	return bashCompleteInstanceNames(cmd)
}
```

@AkihiroSuda I don't quite understand why we have a separate wrapper around `bashCompleteInstanceNames` for each command; why don't we share a single implementation?

Reply:
We can just share the single impl